### ISSUE TITLE
v5.0

### ISSUE THREAD

**Issue by strarsis** [johnpbloch/wordpress#37]
[WordPress 5.0 is out](https://wordpress.org/news/2018/12/bebo/) but I can't find its package release on [packagist](https://packagist.org/packages/johnpbloch/wordpress).

**Comment by nextgenthemes** [comment]
There is a bot that is supposed to update it every 15 minutes. That bot does not seem to be open source or public at least my search for it turned nothing up. I think that is pretty bad as no one else can investigate and fix this. WP.org should have its own official composer package that is just updated as everything else.

**Comment by archon810** [comment]
https://twitter.com/ArtemR/status/1070927328409116672

I asked @m, we'll see if he responds.

**Comment by kraftner** [comment]
@nextgenthemes AFAIK it's this: https://github.com/johnpbloch/build-wp

**Comment by krishaamer** [comment]
Any update on this @johnpbloch?

**Comment by koendoit** [comment]
He is from Irving, TX, think he is sleeping :)

**Comment by maliksvd** [comment]
This is a weird delay and it's problematic.

**Comment by retlehs** [comment]
please stop adding comments to this topic. you realize that this is an open-source project that someone volunteers their free time to manage, right?

pretty sure that it's already clear that there's an issue. no need to add more noise. ✌️

**Comment by johnpbloch** [comment]
I'll get to this when I'm able. It may be as much as a number of days.

**Comment by nextgenthemes** [comment]
I agree with that pinging him multiple times (while he sleeps) would be annoying to me but if this takes multiple days I think it should be OK to talk about it here.

The probably easiest and most straight forward solution to get 5.0 running with this is this in package.json:

```
        "johnpbloch/wordpress": "5.0.x-dev",
	"johnpbloch/wordpress-core": "5.0.x-dev",
```

But it will install the dev version and that may not be what you want. At this stage its probably more or less the same then 5.0. I would guess without looking it up. I think it pull the 5.0 branch from his repo and that was updated 2 days ago to its like RC3/5.0.

So I am relatively new to composer. But I think all what is needed might be just the wp.org guys have a composer.json at the root of https://github.com/WordPress/WordPress (I guess we can w8 years for that to happen). The package itself does not need to be listed on packagist. You can put a custom section on your `composer.json` and reference a Github repo and then tags can be installed, his bot is doing a little more then that is seems but as a workaround for now or just a less complicated solution I think basically forking WP and putting a package.json into it might work unless I am mistaken.

@retlehs Trellis does some specific stuff so just ssh into the server and unpacking the latest release does that work? I guess I just w8 till its ready but imagine this to be critical security release. I think its concerning and I do not mean this in a demanding way its just that people who have normal installs get releases auto installed and we are stuck and dependent on this and it feels bad.

**Comment by johnpbloch** [comment]
5.0 just got built and pushed. The root cause was that I'd added code to skip trying to commit, tag, and push when there were no changes made to the code. The command checking for changes to the repo didn't take untracked files into consideration. That's fixed now. Thanks again everybody for your patience.

For the record, I didn't think this thread was noisy or disrespectful of any boundaries or ungrateful for any OS contributions. I commented as soon as I saw the notifications--it took a while for me to even see them because I was travelling and preoccupied with family concerns. Again, I appreciate the patience while I fixed the issue. 🙇

**Comment by archon810** [comment]
Thanks @johnpbloch!

**Comment by SeBsZ** [comment]
Hey @johnpbloch, adding here because this issue may be related. Wordpress 5.1 was released yesterday, but it seems 5.1 isn't on packagist yet.

**Comment by archon810** [comment]
5.1.0 is there now \o/
